jenkins/scripts/integration_test.sh

```diff
@@ -54,6 +54,10 @@ KUBERNETES_VERSION_UPGRADE_FROM="${KUBERNETES_VERSION_UPGRADE_FROM:-}"
 KUBERNETES_VERSION_UPGRADE_TO="${KUBERNETES_VERSION_UPGRADE_TO:-}"
 KUBECTL_SHA256="${KUBECTL_SHA256:-}"
 
+if [[ "${REPO_NAME}" == "ironic-image" && "${REPO_BRANCH}" == "main" ]]; then
+  IRONIC_INSTALL_TYPE="source"
```

Comment:
Confused which one is used later on `IRONIC_INSTALL_TYPE` or `IRONIC_FROM_SOURCE`

Reply:
Setting `IRONIC_INSTALL_TYPE="source"` here will allow it to be handled properly later on, without having to duplicate the logic.